yarn.lock

```diff
@@ -10917,7 +10917,7 @@ rechoir@^0.6.2:
   dependencies:
     resolve "^1.1.6"
 
-"reconnecting-websocket@github:wireapp/reconnecting-websocket#96ac796ae8c7a7d9d857f46ed39dbedd7b9ad378":
```

Comment:
Please update your yarn installation :)

Reply:
I am running on v1.21.1 which is the latest.

Reply:
:thinking: okay